can-define-class.js

```diff
@@ -62,13 +64,22 @@ function getKeyValue(key) {
 	}
 }
 
+function defineNonEnumerable(obj, prop, value) {
```

Comment:
Is this any different from the one exported by define.js? If not can we just use that one? https://github.com/canjs/can-define-class/blob/bf83944ef3499f64e9e01974a30e468af8505be9/define.js#L1145